frappe/public/js/frappe/list/list_view.js

```diff
@@ -1494,6 +1494,14 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 			});
 		}
 
+		items.push({
```

Comment:
we can follow the same suit for this as importing? 🤔 - just redirect to data export

we also have `can_export` function in `model.js` - adding a check for it would be much saner?

https://github.com/nikkothari22/frappe/blob/7a89893dcb35cefa221be5d5f396f5a564989a28/frappe/public/js/frappe/model/model.js#L370

#

reason i'm kinda skeptical is:
a) permissions
b) the bundle requirement of data_import in list view

Reply:
The Data Export page does not export it in a format that allows us to import it directly in another Frappe site. (Refer: https://github.com/frappe/frappe/pull/18241#issuecomment-1260409262)

For the permissions, I checked the python function in exporter.py - it checks for permissions - whether the user can export or not. So it will raise an error if the user does not have permissions.

The data_import file only imports two things: 'import_preview' and 'data_exporter'. I agree we only need 'data_exporter' here - but is there a way to refactor it or is it too much work?

Reply:
@nikkothari22 Current Data Export tool needs to go. It outputs an unusable format. It would be nice if Exporter from Data Import tool is refactored so that it can be used at multiple places like this one.

Reply:
![Screenshot 2022-10-19 at 10 53 26 AM](https://user-images.githubusercontent.com/9079960/196604292-1e3c642e-8554-4d35-bc21-762f3b9c46c7.png)


btw data export is already present on list view. It requries selecting items to export first thogh.